channeldb/invoices.go

```diff
@@ -1170,6 +1170,7 @@ func copyInvoice(src *Invoice) *Invoice {
 		Receipt:        copySlice(src.Receipt),
 		PaymentRequest: copySlice(src.PaymentRequest),
 		FinalCltvDelta: src.FinalCltvDelta,
+		Expiry:         src.Expiry,
```

Comment:
Commit doesn't adhere to the contribution guidelines section about commit message crafting: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#45-ideal-git-commit-structure

Reply:
done